docs/csharp/misc/cs0460.md

```diff
@@ -14,9 +15,14 @@ Constraints for override and explicit interface implementation methods are inher
   
  When a generic method that is part of a derived class overrides a method in the base class, you may not specify constraints on the overridden method. The override method in the derived class inherits its constraints from the method in the base class.  
   
+However, there are exceptions to this rule:
+
+- Starting with C# 9, you can apply the `default` constraint to override and explicit interface implementation methods to resolve ambiguities with nullable reference types.  
```

Comment:
```suggestion
- Starting with C# 9, you can apply the `default` constraint to `override` and explicit interface implementation methods to resolve ambiguities with nullable reference types.  
```

Reply:
When I first read it, I read it as "apply the constraint to override ...", as in 'override' is the action you want to take. So maybe the backticks help.